.github/workflows/auto_cherry_pick.yml

```diff
@@ -43,3 +43,37 @@ jobs:
           labels: |
             Auto_Cherry_Picked
             ${{ matrix.to_branch }}
+      - name: Set issue title
```

Comment:
We should create a separate job that creates an issue depending on the auto-cherry-pick job. That would nicely separate it from cherry-picking .

Hint: Use `needs` directive to point to `auto-cherry-pick` job and if conditioning of that jobs success if fails!